.github/workflows/config/labeler.yml

```diff
@@ -1,33 +1,427 @@
-# Configuration for labeler - https://github.com/actions/labeler
-
-documentation:
-- "*/manifest.json"
-- "*/assets/configuration/**"
-- (?<!CHANGELOG)\.md$
-- docs/**
-
+changelog/no-changelog:
+- requirements-agent-release.txt
+- datadog_checks_dev/datadog_checks/dev/__about__.py
+- all:^(?![^/]+/datadog_checks/.*|[^/]+/pyproject\.toml).*
+dependencies:
+- datadog_checks_base/datadog_checks/base/data/agent_requirements.in
 dev/testing:
 - .azure-pipelines/**
 - .codecov.yml
-
 dev/tooling:
 - .github/workflows/**
 - .github/stale.yml
 - .gitlab/**
 - .gitlab-ci.yml
 - datadog_checks_dev/datadog_checks/dev/tooling/**
-
-dependencies:
-- datadog_checks_base/datadog_checks/base/data/agent_requirements.in
-
+documentation:
+- '*/manifest.json'
+- '*/assets/configuration/**'
+- (?<!CHANGELOG)\.md$
+- docs/**
+integration/active_directory:
+- active_directory/**/*
+integration/activemq:
+- activemq/**/*
+integration/activemq_xml:
+- activemq_xml/**/*
+integration/aerospike:
+- aerospike/**/*
+integration/agent_metrics:
+- agent_metrics/**/*
+integration/airflow:
+- airflow/**/*
+integration/amazon_eks:
+- amazon_eks/**/*
+integration/amazon_eks_blueprints:
+- amazon_eks_blueprints/**/*
+integration/amazon_msk:
+- amazon_msk/**/*
+integration/ambari:
+- ambari/**/*
+integration/apache:
+- apache/**/*
+integration/arangodb:
+- arangodb/**/*
+integration/aspdotnet:
+- aspdotnet/**/*
+integration/avi_vantage:
+- avi_vantage/**/*
+integration/azure_active_directory:
+- azure_active_directory/**/*
+integration/azure_iot_edge:
+- azure_iot_edge/**/*
+integration/boundary:
+- boundary/**/*
+integration/btrfs:
+- btrfs/**/*
+integration/cacti:
+- cacti/**/*
+integration/calico:
+- calico/**/*
+integration/cassandra:
+- cassandra/**/*
+integration/cassandra_nodetool:
+- cassandra_nodetool/**/*
+integration/ceph:
+- ceph/**/*
+integration/cilium:
+- cilium/**/*
+integration/cisco_aci:
+- cisco_aci/**/*
+integration/citrix_hypervisor:
+- citrix_hypervisor/**/*
+integration/clickhouse:
+- clickhouse/**/*
+integration/cloud_foundry_api:
+- cloud_foundry_api/**/*
+integration/cockroachdb:
+- cockroachdb/**/*
+integration/confluent_platform:
+- confluent_platform/**/*
+integration/consul:
+- consul/**/*
+integration/consul_connect:
+- consul_connect/**/*
+integration/container:
+- container/**/*
+integration/containerd:
+- containerd/**/*
+integration/coredns:
+- coredns/**/*
+integration/couch:
+- couch/**/*
+integration/couchbase:
+- couchbase/**/*
+integration/cri:
+- cri/**/*
+integration/crio:
+- crio/**/*
+integration/databricks:
+- databricks/**/*
+integration/datadog_checks_base:
+- datadog_checks_base/**/*
+integration/datadog_checks_dependency_provider:
+- datadog_checks_dependency_provider/**/*
+integration/datadog_checks_dev:
+- datadog_checks_dev/**/*
+integration/datadog_checks_downloader:
+- datadog_checks_downloader/**/*
+integration/datadog_checks_tests_helper:
+- datadog_checks_tests_helper/**/*
```

Comment:
While we're here, WDYTA finally removing `integration/` since they aren't integrations?

Reply:
Well actually `datadog_checks_dependency_provider` is

Reply:
Good idea, we can do that

Reply:
I just removed the `integration/` prefix and made each of those packages their own label. Does that look ok, or should their labels go under a different prefix, like `dev/`?